[WIP] Theory and Practice of Working Groups - Community Handbook (#3499)
the-turing-way/the-turing-way: book/website/community-handbook/governance/working-groups.md

- **Secretary**: This person is responsible for ensuring the logistical reporting of the group. They are responsible for uploading meeting notes, ensuring that time-keeping is set in meetings, and that records of the group's activities are transparent, open, and accessible for others to access. For example, if the duties of the chair are distributed across multiple people, the secretary is responsible for documenting this format in the working group's repository.
- **Turing Staff Delivery Team Contact**: While working groups should be able to operate with autonomy in most decisions that apply to their work, sometimes they may require help from the TTW core staff delivery team in order to enable their operations, and/or support the incubation of the group. Each working group should have an assigned contact from the staff delivery team, most likely either the project Community Manager or Project Manager.

> - **Member**: Any Turing Way community member may join a working group without restriction, and without taking on a leadership role.

Reviewer: I have edited the file to rephrase this part.

* **Create a working group charter**: [TEMPLATE LINKED HERE](https://hackmd.io/@aleesteele/template-ttw-wg-charter)
    * * Note: All working groups should upload this charter to their respective github repository [in The Turing Way organisation](https://github.com/the-turing-way/the-turing-way/).
* **Create an open calendar invite**: This may require support of a core delivery staff member to add the information to the public google calendar
* **Advertise on Community channels**:  Slack, Github, social media and/or the TTW newsletter to share the information about your working group
    * Note: The Community Manager and/or Project Manager can help with this! Ask them on Slack or tag them on Github to share out this information, or co-create a communication plan.
* **Create a slack channel for collaboration** (not required - but encouraged): Having an ongoing channel makes it easier to communicate across. What other communication channels might this group need?
* **Add information about working group in Ways of Working and/or Governance documents**: Is there a public and up-to-date record of this working group?

### Approving a Working Group

> The approval process for a working group requires a bit more involvement from the wider community and leadership council, as it is important to understand what resources might be available to supp, and/or if the working group's aims are appropriate for the community. 

Reviewer 2: ```suggestion
The approval process for a working group requires a bit more involvement from the wider community and leadership council. 
It is important to understand what resources might be available to support the working group, and/or if the working group's aims are appropriate for the community. 
```

1. **Completed community charter**: All working groups need to fill out a proposed working group charter ([template here](https://hackmd.io/@aleesteele/template-ttw-wg-charter)) that should be presented to the TTW leadership council and broader community for discusssion and review

> 2. **Open discussion and/or Q&A**: All Discuss your idea in community spaces: such as the TTW Slack workspace, Github discussions/issues, and/or Community Calls like the Collaboration Cafe.  

Reviewer 2: ```suggestion
2. **Open discussion and/or Q&A**: Discuss your idea in community spaces: such as the TTW Slack workspace, GitHub discussions/issues, and/or Community Calls like the Collaboration Cafe.  
```

> 3. **Leadership council approval**: This creation of this working group must be approved by *The Turing Way* leadership council at their regular organisating calls. These meetings happen every three months, which you can find out about [in this schedule](https://).

Reviewer 2: ```suggestion
3. **Leadership council approval**: This creation of this working group must be approved by *The Turing Way* leadership council at their regular organisating calls. 
These meetings happen every three months, which you can find out about [in this schedule](https://).
```

### Maintaining a Working Group

While initiating a working group, maintaining a working group's activities is integral for the group's success. 

> On a quarterly basis, a working group should revisit its structures and ways of working in order to establish that the group is adhering to its established purpose. The following questions can be guidelines for doing so. These questions have also been compiled into this [working group quarterly audit template]():

Reviewer 2: ```suggestion
On a quarterly basis, a working group should revisit its structures and ways of working in order to establish that the group is adhering to its established purpose. 
The following questions can be guidelines for doing so. 
These questions have also been compiled into this [working group quarterly audit template]():
```

* **Review purpose**: Is this working group still fulfilling its established purpose? 

> * **Review progress**: Has this working been able to make progress towards its stated goals? What adjustments may need to be made in order to support this kind of work?

Reviewer 2: ```suggestion
* **Review progress**: Has this working been able to make progress towards its stated goals? 
What adjustments may need to be made in order to support this kind of work?
```

> * **Review process**: Is the existing way of working valuable for this team currently? Are there other processes (decision-making, conflict manage)

Reviewer 2: ```suggestion
* **Review process**: Is the existing way of working valuable for this team currently? 
Are there other processes (decision-making, conflict management)
```

    * Note: The Community Manager and/or Project Manager can help with facilitating this! 
* **Review roles**: Is everyone in the group in a role they would like to stay in? If not, how should these roles change?

### Sunsetting a Working Group

> Deciding when to sunset a working group can be difficult. Often, working groups may continue on indefinitely, and while there are a variety of reasons for this, this may be because there are not means of recognising when a group's aims and/or goals have shifted, or if a group's work is no longer needed. Knowing when it might be time to sunset a working group or shift its aims or direction is an important

Reviewer 2: ```suggestion
Deciding when to sunset a working group can be difficult. 
Often, working groups may continue on indefinitely, and while there are a variety of reasons for this, this may be because there are not means of recognising when a group's aims and/or goals have shifted, or if a group's work is no longer needed. 
```

> Reasons for sunsetting a working group can be for the following

Reviewer 2: ```suggestion
Reasons for sunsetting a working group can be for the following:
```

> * **Decreased capacity**: Are there no longer enough people to support this working group? If the number of peope able to sustain a working group decreases to below two, the possibility of shutting down the working group should be discussed.

Reviewer 2: ```suggestion
* **Decreased capacity**: Are there no longer enough people to support this working group? 
If the number of people able to sustain a working group decreases to below two, shutting down the working group should be discussed.
```

* **Disciplinary action**: If there have been any actions taken that may have broken *The Turing Way* code of conduct, the Staff Delivery Team and/or TTW way co-leads reserve the right to retire the working group.

> * **Decision from leadership council**: If the actions or projects realted to the working group are not seen to be relevant to *The Turing Way* project more broadly, it is possible for the leadership council to disban a working group through a voting process.

Reviewer 2: ```suggestion
* **Decision from leadership council**: If the actions or projects related to the working group are not seen to be relevant to *The Turing Way* project more broadly, it is possible for the leadership council to disband a working group through a voting process.
```

Once a group has decided to close down its efforts, consider the following:
* **Document reasons for closing down working group**: Documentation is important for continuous learnings - for future and current working groups, as well as the wider community and even ecosystem that The Turing Way feeds into!
    * If you need help with this documentation process, check out this [template]()

> * **Change the meeting information**: Remember to cancel meeting invites, and archive existing notes in the github repository

Reviewer 2: ```suggestion
* **Change the meeting information**: Remember to cancel meeting invites, and archive existing notes in the GitHub repository
```
